main.py

```diff
@@ -2,48 +2,62 @@
 # Encoding: Utf-8
 import random
 from collections import OrderedDict, deque
-import keys  # create a keys.py file with your twitter tokens if you want to run your own instance !
+# create a keys.py file with your twitter tokens if you want to run your own
+# instance !
+import keys
 import logging
 import json
 from TwitterAPI import TwitterAPI, TwitterRequestError
 
 logging.basicConfig(filename="/tmp/coffebot.log", level=logging.INFO)
 
-coffeeFile  = open("coffeeType.json")
+coffeeFile = open("coffeeType.json")
```

Comment:
Some variable names use `mixedCase` and some use `snake_case`. Should all be `snake_case` as per Python PEP8 style conventions.

https://www.python.org/dev/peps/pep-0008/#naming-conventions